src/App.tsx

```diff
@@ -1,8 +1,26 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from 'react';
+import './App.scss';
+import Search from './components/Search/index';
+import getSearch from './api/searchCities';
 
-function App() {
-    return <div className="App" />;
+function App(): JSX.Element {
+    const [search, setSearch] = useState<string>('');
+    const [items, setItems] = useState<string[]>([]);
```

Comment:
this is parent component and there will be many different arrays and will you name them as items1, items2, items3, or will give them meaningful names, such as cities, favoriteCities, users, weathers and so on?